lib/eunit/test/eunit_SUITE.erl

```diff
@@ -203,3 +208,40 @@ check_surefire(Module) ->
 	%% Check that file is valid XML
 	xmerl_scan:file(File),
 	Chars.
+
+scale_timeouts_test(_Config) ->
+    %% Scaling with integers
+    Millis1 = run_eunit_test_that_times_out(times_out_test_,
+                                            [{scale_timeouts, 2}]),
+    ?assert(Millis1 >= 2000, #{duration => Millis1}),
```

Comment:
- why 2000 and not 20 000 (or at least >10 000) ?
- maybe worth checking Millis1 is less than something ?

Reply:
The test times out after 1 second, and it is scaled up with a factor of 2, so it should take at least 2 seconds for it to complete. (the test with a 1-second timeout sleeps for a long time, 10s, to make sure it times out.)

I added some comments that hopefully clarifies things a bit.

Checking whether it does not sleep too long: Ok. I was a bit afraid of making a flaky test, for example if it runs on a server, maybe a virtual host, and there is some hiccup of any kind. So I thought I can test for a lower bound, but need some margin for an upper bound. That's was my reasoning for setting fairly relaxed upper bounds checks, but I lowered them a bit after your comment.

Reply:
you might be correct, but let us at least check how it behaves before merging. I will re-apply testing label after OTP-26.1 is released.